styles.css

```diff
@@ -29,7 +29,120 @@ a {
 ul.nolist {
 	list-style-type: none;
 	margin: 0;
-	padding: 5px 15px;
+	padding: 0;
+}
+
+.button {
+	display: inline-block;
+	text-transform: uppercase;
+	font-size: 10px;
+}
+
+.button:after {
+	content: "";
+	display: inline-block;
+	margin: 0 5px;
+}
+
+.button:last-of-type:after {
+	margin: 0;
+	display: none;
+}
+
+/* Form Elements */
+
+.form-control {
+	border: 4px solid silver;
+	font-size: 20px;
+	outline: none;
+	padding: 20px 10px;
+}
+
+.form-control:focus {
+	border-color: gray;
+}
+
+select.form-control {
+	height: 48px;
+}
+
+.form-inline .form-group {
+	vertical-align: top;
+	margin-right: 10px;
+}
+
+label {
+	display: block;
+}
+
+.error-message {
+	color: red;
+	font-size:;
+}
+
+.form-radio label:first-of-type {
+	margin-bottom: 15px;
+}
+
+.form-radio input[type="radio"] {
+	padding: 0;
+	display: none;
+}
+
+.form-radio fieldset label {
+	display: inline-block;
+  cursor: pointer;
+  position: relative;
+  padding-left: 35px;
+  margin-right: 15px;
+  font-size: 13px;
+}
+
+.form-radio fieldset label:before {
+	content: "";
+  display: inline-block;
+  width: 25px;
+  height: 25px;
+  margin-right: 10px;
+  position: absolute;
+  left: 0;
+  bottom: -2px;
+  background-color: silver;
+  border: 4px solid silver;
+  border-radius: 25px;
+}
+
+input[type="radio"]:checked + label:before {
+  content: "";
+  background-color: gray;
+  border-color: gray;
+  color: #f3f3f3;
+  text-align: center;
+}
```

Comment:
And here's what makes it work. Clicking on a label in a browser (if the `for="fieldname"` is set) will automatically place the input cursor in the field. For radio buttons, this "checks" the button. So clicking the `label:before` actually selects the radio button it corresponds to!

The `+ label:before` does this: If the radio button has the status of "checked" (ie: clicked), it will change the style of the `label:before` to fill in with a darker color, tricking the user into seeing the radio button as filled.

![MAGIC](http://i.imgur.com/rAlU7Ku.gif)